src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -327,24 +327,27 @@ public FunctionDescriptor GetHttpFunctionOrNull(Uri uri)
             return function;
         }
 
-        protected override void OnInitializeConfig(JobHostConfiguration config)
+        protected override void OnInitializeConfig(ScriptHostConfiguration config)
         {
+            config.TraceWriter = new SystemTraceWriter(TraceLevel.Verbose);
```

Comment:
This system trace writer gets wrapped up into a composite that is passed throughout the runtime, ensuring that we can trace everything. It will also trace all core SDK logs as well.